internal/database/mariadb/database.go

```diff
@@ -73,7 +74,8 @@ func NewSqlDatabase(cfg util.Config) (*SqlDatabase, error) {
 	}
 	db.Exec(fmt.Sprintf("USE %s", cfg.DBName))
 	return &SqlDatabase{
-		db: db,
+		db:     db,
+		config: &cfg,
```

Comment:
Do we want to pass the whole config to the database? It does not only contain database configuration WDY @drochow @dorneanu